src/interfaces/Server/IServerJobs.ts

```diff
@@ -11,10 +11,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { TypeDaprJobsCallback } from "../../types/DaprJobsCallback.type";
-
 export default interface IServerJobs {
-  listen(jobName: string, callback: TypeDaprJobsCallback): void;
-
-  listen<DataType>(jobName: string, callback: TypeDaprJobsCallback<DataType>): void;
+  listen(jobName: string, callback: (data: any) => unknown): void;
```

Comment:
Why does this return an `unknown`? While the callback needn't necessarily return anything, the SDK does need to know that it was handled without error so it can send back a 200 OK.